04-Time-Series-Decomposition/06-MSTL-decomposition.ipynb

```diff
@@ -652,7 +652,8 @@
     "ax_last.xaxis.set_ticks(pd.date_range(start=\"2012-01-01\", freq=\"MS\", periods=5))\n",
     "plt.setp(ax_last.get_xticklabels(), rotation=0, horizontalalignment=\"center\")\n",
     "for ax in axs[:-1]:\n",
-    "    ax.get_shared_x_axes().join(ax, ax_last)\n",
+    "    if not ax.get_shared_x_axes().joined(ax, ax_last):\n",
+    "        ax.sharex(ax_last)\n",
```

Comment:
I'd just add line 656 and remove 655, if the code works in the current pandas version. Could you update it @ganeshkamath89 ?

Reply:
Hi @solegalli. I tried the changes. 
The plot looks the same before and after the change.
Thanks for reviewing the commit.

Before commite:
![image](https://github.com/trainindata/feature-engineering-for-time-series-forecasting/assets/2648522/b8e079c3-9f29-4bc4-94ed-bd65fe6b925e)

After commit:
![image](https://github.com/trainindata/feature-engineering-for-time-series-forecasting/assets/2648522/1be8aa02-cf67-46ac-8631-ac8e92c71e47)